src/utils/absoluteUrl.test.js

```diff
@@ -0,0 +1,49 @@
+import absoluteUrl from './absoluteUrl';
+
+describe('absoluteUrl', () => {
+  test('should return /path_1/path_2/path_3/path_to_destination when the window.location.origin is http://dummy.com/path_1/path_2 and the path is /path_3/path_to_destination', () => {
+    global.window = Object.create(window);
+    const url = 'http://dummy.com/path_1/path_2';
+    Object.defineProperty(window, 'location', {
+      value: {
+        origin: url,
+      },
+      writable: true,
+    });
+    const expectedAbsoluteUrl = absoluteUrl('/path_3/path_to_destination');
+    expect(expectedAbsoluteUrl).toEqual(
+      '/path_1/path_2/path_3/path_to_destination'
```

Comment:
```
const expectedAbsoluteUrl = '/path_3/path_to_destination';
const result = absoluteUrl('/path_1/path_2/path_3/path_to_destination');

expect(result).toEqual(expectedAbsoluteUrl);
```

I think this is the intended phrasing/setup?

Reply:
I didn't get your question.

Reply:
I'm suggesting that we switch around the variable names so it's easier to follow what we're asserting.

The "ExpectedAbsoluteUrl" should probably be the known good value, not the "unknown" value returned by our "system under test"

IE. `expect result to equal our known good value`

Reads easier than what we currently have:

`expect expectedAbsoluteUrl to equal some string value`

Reply:
done